docs/content/apis/dmf_api.md

```diff
@@ -52,12 +52,22 @@ Message Properties | Description
 content_type       | The content type of the payload                                                                      | String | true
 reply_to           | Exchange to reply to. The default is sp.direct.exchange which is bound to the sp_direct_queue | String | false
 
-Example headers
+Example headers and payload:
 
 Header                                                                             | MessageProperties
 ----------------------------------------------------------------------------------- | -------------------------------------------------------------------------------
 type=THING\_CREATED <br /> tenant=tenant123 <br /> thingId=abc  <br /> sender=Lwm2m | content\_type=application/json <br /> reply_to (optional) =sp.connector.replyTo
 
+Payload Template (optional):
+
+```json
+{
+    "name": "String"
+}
+```
+
+The "name" property specifies the name of the thing, which per default is the thing ID. This property is optional.
+
```

Comment:
move this template description above the Example section and add a real example for the payload here. 
e.g. { "name" : "My Precious Thing" }

Reply:
Or maybe not. 
General question: Is there a reason why the doc structure of (Client -> hawkBit) differs from the one of (hawkBit -> Client)?

Reply:
I am not aware of a dedicated reason, but it was like this for years so I would not change it.

Reply:
Okay